ex/lib/node/node_proto.ex

```diff
@@ -175,63 +175,4 @@ defmodule NodeProto do
   def unpack_message_v2(data) do
     unpack_message(data)
```

Comment:
this wont exist no types to tell it so tho so hard to notice.

can just null it so it doesnt crash every invalid match above?